manager/probe.go

```diff
@@ -507,15 +509,17 @@ func (p *Probe) stop(saveStopError bool) error {
 	err := p.detachRetry()
 
 	// close the loaded program
-	err = ConcatErrors(err, p.program.Close())
+	if p.attachRetryAttempt >= p.ProbeRetry {
```

Comment:
Doesn't this mean it won't close the program if I call `Probe.Stop()`?

Reply:
Fixed by setting `p.attachRetryAttempt = p.ProbeRetry` when the attach is successful.